src/transport_ofi.c

```diff
@@ -292,13 +292,6 @@ static inline int allocate_endpoints(struct fabric_info *info)
 
     /* this endpoint is used only to get read and write
      * counter updates */
-    info->p_info->caps = FI_RMA | FI_WRITE | FI_READ | /*SEND ONLY */
-                         FI_ATOMICS; /* request atomics capability */
```

Comment:
The comment indicates that the intent here to have a TX only endpoint. However, the caps indicate RX capabilities as well (also follow on bits that are OR'd in).

I'd like to understand better what the intent is, and maybe what we need here is to downselect some RX attributes from the endpoint that is being created.